pkg/controller/login/reset_password.go

```diff
@@ -73,8 +74,14 @@ func (c *Controller) HandleSubmitResetPassword() http.Handler {
 				return
 			}
 
-			controller.InternalError(w, r, c.h, err)
-			return
+			// Re-create the firebase user if it does not exist.
+			// We have seen this disappear if a user never logs-on after being invited
+			if created, _ := c.authProvider.CreateUser(ctx, user.Name, user.Email, "", false, nil); created {
+				stats.Record(ctx, controller.MFirebaseRecreates.M(1))
```

Comment:
This will fire in dev, so I don't think it's what we want. I'll add something to the other PR

Reply:
we probably don't want an alert on single instances, but if we see a rash over these over a time period, that seems noteworthy

Reply:
I don't think we alert on this metric?

Reply:
but also feel free to cannibalize this change

Reply:
I see. We should move this into the login controller's metrics then. I can do that as part of the other PR if that's easier.

Reply:
Pulled into https://github.com/google/exposure-notifications-verification-server/pull/1606

/close